packages/next-admin/src/utils/props.ts

```diff
@@ -154,15 +155,17 @@ export async function getPropsFromParams({
       const edit = options?.model?.[resource]?.edit as EditOptions<
         typeof resource
       >;
-      schema = transformSchema(schema, resource, edit);
-      schema = await fillRelationInSchema(
-        schema,
+      
+      let deepCopySchema = cloneDeep(schema);
```

Comment:
Why are we doing this ?

Reply:
During some operations we make some changes to `schema`, if these changes are made to original `schema` they will persist for next operations. By cloning `schema` we can preserve the original `schema`.